build: Fail to cross-compile depends for HOST=x86_64-apple-darwin16 on Focal

System: Linux Mint 20 (focal codebase, x86_64).
```
$ make -C depends HOST=x86_64-apple-darwin16
...
Extracting native_cctools...
/home/hebasto/GitHub/bitcoin/depends/sources/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4.tar.gz: OK
/home/hebasto/GitHub/bitcoin/depends/sources/clang-llvm-8.0.0-x86_64-linux-gnu-ubuntu-14.04.tar.xz: OK
/home/hebasto/GitHub/bitcoin/depends/sources/3efb201881e7a76a21e0554906cf306432539cef.tar.gz: OK
Preprocessing native_cctools...
CMake Deprecation Warning at CMakeLists.txt:14 (cmake_policy):
  The OLD behavior for policy CMP0051 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.


-- The C compiler identification is unknown
-- The CXX compiler identification is unknown
-- The ASM compiler identification is unknown
-- Found assembler: /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/toolchain/bin/clang
-- Check for working C compiler: /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/toolchain/bin/clang
-- Check for working C compiler: /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/toolchain/bin/clang -- broken
CMake Error at /usr/share/cmake-3.16/Modules/CMakeTestCCompiler.cmake:60 (message):
  The C compiler

    "/home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/toolchain/bin/clang"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeTmp
    
    Run Build Command(s):/usr/bin/make cmTC_6d157/fast && make[1]: Entering directory '/home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeTmp'
    /usr/bin/make -f CMakeFiles/cmTC_6d157.dir/build.make CMakeFiles/cmTC_6d157.dir/build
    make[2]: Entering directory '/home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeTmp'
    Building C object CMakeFiles/cmTC_6d157.dir/testCCompiler.c.o
    /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/toolchain/bin/clang    -o CMakeFiles/cmTC_6d157.dir/testCCompiler.c.o   -c /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeTmp/testCCompiler.c
    /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/toolchain/bin/clang: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory
    make[2]: *** [CMakeFiles/cmTC_6d157.dir/build.make:66: CMakeFiles/cmTC_6d157.dir/testCCompiler.c.o] Error 127
    make[2]: Leaving directory '/home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeTmp'
    make[1]: *** [Makefile:121: cmTC_6d157/fast] Error 2
    make[1]: Leaving directory '/home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeTmp'
    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:45 (project)


-- Configuring incomplete, errors occurred!
See also "/home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeOutput.log".
See also "/home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/libtapi/build/CMakeFiles/CMakeError.log".
make: *** [funcs.mk:263: /home/hebasto/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/native_cctools/4da2f3b485bcf4cef526f30c0b8c0bcda99cdbb4-cc2135bcaca/.stamp_preprocessed] Error 1
make: Leaving directory '/home/hebasto/GitHub/bitcoin/depends'
```

--- a/depends/README.md
+++ b/depends/README.md
@@ -44,7 +44,7 @@ The paths are automatically configured and no other options are needed unless ta
 
 #### For macOS cross compilation
 
-    sudo apt-get install curl librsvg2-bin libtiff-tools bsdmainutils cmake imagemagick libcap-dev libz-dev libbz2-dev python3-setuptools
+    sudo apt-get install curl librsvg2-bin libtiff-tools bsdmainutils cmake imagemagick libcap-dev libz-dev libbz2-dev python3-setuptools libtinfo5
 
 #### For Win64 cross compilation
 

